bin/rails

```diff
@@ -8,13 +8,31 @@ require_relative '../config/boot'
 # - useful when running inside a container
 require 'rails/command'
 require 'rails/commands/server/server_command'
+module Baw
+  module DefaultOptions
+    # adapted from https://github.com/rails/rails/blob/9b6f7d7f547e22e73f3127999b154a3bab3ff14e/railties/lib/rails/commands/server/server_command.rb#L220
+    def host
+      if options[:binding]
+        options[:binding]
+      else
+        # i changed only the following line
+        default_host = '0.0.0.0'
 
-module DefaultOptions
-  def default_options
-    super.merge!(Host: '0.0.0.0')
+        if ENV['HOST'] && !ENV['BINDING']
+          ActiveSupport::Deprecation.warn(<<-MSG.squish)
+          Using the `HOST` environment variable to specify the IP is deprecated and will be removed in Rails 6.1.
+          Please use `BINDING` environment variable instead.
+          MSG
+
+          return ENV['HOST']
+        end
```

Comment:
Is there a reason the old method is still supported considering we are upgrading to 6.1? Considering we are using a docker container this should be under our control no?

Reply:
Oh yeah, the old monkey patch was no longer effective on rails 6.1. I just hadn't noticed until now.

Reply:
Oh and to clarify, this patch changes the default behaviour for the options for `rails s'. We could always force a behaviour by setting an ENV var in the docker host, but that would prevent and of the CLI switches from working if we wanted any different behaviour (say running two versions of the web server for testing).

Changing the default option seems like the least intrusive choice